plugins/zenoh-plugin-rest/examples/z_serve_sse.rs

```diff
@@ -46,7 +47,7 @@ async fn main() {
     let queryable = session.declare_queryable(key).res().await.unwrap();
 
     async_std::task::spawn({
-        let receiver = queryable.receiver.clone();
+        let receiver = queryable.deref().clone();
```

Comment:
I find odd for the user to actually call `.deref()` to get the `Handler` of the subscriber.
In this particular case, the default `Handler` is a `flume::Receiver` but it could be something else.

IMHO we should add `.handler()` and `.handler_mut()` methods to `subscriber`, `queryable`, etc. that allows the user to explicitly retrieve the actual handler.

Reply:
As discussed offline with @DenisBiryukov91 and https://github.com/wyfo, I've added `.handler()` and `.handler_mut()` to both `subscriber` and `queryable`.